app/services/report_importer.rb

```diff
@@ -44,6 +44,15 @@ def import
     end
   end
 
+  def scan(save = false)
+    logger.info "Scanning report with: #{report_scanner.join(', ')}"
+    report_scanner.each do |scanner|
+      scanner.scan(report)
+      logger.debug "Changes after #{scanner}: #{report.changes.inspect}"
+    end
+    report.save if save
```

Comment:
so this method now returns true/false/nil, should it rather return return value of `scanner.scan(report)` which returns 'Puppet' or nil?